wings/radar_relay_market.pyx

```diff
@@ -339,6 +340,30 @@ cdef class RadarRelayMarket(MarketBase):
                 self._trading_rules[trading_rule.symbol] = trading_rule
             self._last_update_trading_rules_timestamp = current_timestamp
 
+    async def get_account_orders(self) -> List[Dict[str, Any]]:
+        list_account_orders_url = f"{RADAR_RELAY_REST_ENDPOINT}/accounts/{self._wallet.address}/orders"
+        return await self._api_request(http_method="get", url=list_account_orders_url)
+
+    async def get_order(self,
```

Comment:
Adding the account orders map to `get_order()` here violates the single responsibility principle... Ideally `get_order()` should only get a single order. The logic to ignore already obtained orders should go in `_get_order_updates()`